contracts/ERC20_Bridge_Logic_Restricted.sol

```diff
@@ -112,28 +79,43 @@ contract ERC20_Bridge_Logic_Restricted is IERC20_Bridge_Logic_Restricted {
 
     // depositor => is exempt from deposit limits
     mapping(address => bool) exempt_depositors;
-    address exemption_lister;
+
 
     /// @notice This function sets the lifetime maximum deposit for a given asset
     /// @param asset_source Contract address for given ERC20 token
     /// @param lifetime_limit Deposit limit for a given ethereum address
+    /// @param threshold Withdraw size above which the withdraw delay goes into effect
     /// @param nonce Vega-assigned single-use number that provides replay attack protection
     /// @param signatures Vega-supplied signature bundle of a validator-signed order
     /// @dev asset must first be listed
-    function set_lifetime_deposit_max(address asset_source, uint256 lifetime_limit, uint256 nonce, bytes calldata signatures) public override {
+    function set_asset_limits(address asset_source, uint256 lifetime_limit, uint256 threshold, uint256 nonce, bytes calldata signatures) public override {
       require(listed_tokens[asset_source], "asset not listed");
-      bytes memory message = abi.encode(asset_source, lifetime_limit, nonce, 'set_lifetime_deposit_max');
+      bytes memory message = abi.encode(asset_source, lifetime_limit, threshold, nonce, 'set_asset_limits');
       require(IMultisigControl(multisig_control_address()).verify_signatures(signatures, message, nonce), "bad signatures");
-      asset_deposit_lifetime_limit[asset_source] = lifetime_limit;
+      if (asset_deposit_lifetime_limit[asset_source] != lifetime_limit) {
+	  asset_deposit_lifetime_limit[asset_source] = lifetime_limit;
+      }
+      if (withdraw_thresholds[asset_source] != threshold) {
+	  withdraw_thresholds[asset_source] = threshold;
+      }
```

Comment:
I'm not sure the ifs are necessary here, added them so we don;'t write on the map for no reason, but that may be overkill

Reply:
this is overkill